lib/matplotlib/backends/registry.pyi

```diff
@@ -3,8 +3,8 @@ from types import ModuleType
 
 
 class BackendFilter(Enum):
-    INTERACTIVE: int
-    NON_INTERACTIVE: int
+    INTERACTIVE = 0
+    NON_INTERACTIVE =1
```

Comment:
This is weird to me, we have to redefine the Enum class for type checkers... sigh.

Reply:
This is part of the cost of our choice to use stub files.

Reply:
Missing a space after `=` here.
```suggestion
    INTERACTIVE = 0
    NON_INTERACTIVE = 1
```